src/transformers/modeling_utils.py

```diff
@@ -3917,7 +3925,7 @@ def _fix_key(key):
         elif add_prefix_to_model:
             expected_keys = [".".join([prefix, s]) for s in expected_keys]
 
-        missing_keys = list(set(expected_keys) - set(loaded_keys))
+        missing_keys = sorted(set(expected_keys) - set(loaded_keys))
```

Comment:
this might slow down from pretrained for huge models no?

Reply:
Given that the resulting set will be empty or only contain a few keys most of the time, and even sorting a list 1000s of lines long will take fractions of a second, I'd say it's worth it here for the ease of debugging for the user.